tests/RedshiftDatatypeTest.php

```diff
@@ -33,6 +33,28 @@ public function testValidNumericLengths()
         new Redshift("NUMERIC", ["length" => "37,0"]);
         new Redshift("NUMERIC", ["length" => "37,37"]);
         new Redshift("NUMERIC", ["length" => "37"]);
+        new Redshift('NUMERIC', [
+            'length' => [
+                'numeric_precision' => '37',
+                'numeric_scale' => '0'
+            ]
+        ]);
+        new Redshift('NUMERIC', [
```

Comment:
uz me to zaujalo minule, ale nemely by tady byt nekjaky asserty treba ? :)

Reply:
jj taky jsem se nad tím pozastavil, ale pokud by bylo v těch délkách něco špatně tak by to skončilo `InvalidLengthException`

Reply:
`$this->expectNotToPerformAssertions()`?

Případně tohle udělat jako dataprovider a v testu z instance něco vytahnout a zkontrolovat, že to vrací co má...

Reply:
Každopádně chybí test na to, že 

```
new Redshift('NUMERIC', [
            'length' => [
                'numeric_precision' => '37',
                'numeric_scale' => '0'
            ]
        ]);
$this->assertSame(37, $redshhift->getPrecision());
```

 nebo tak něco.

Reply:
jako taky bych rekl, ze by to tam melo byt, i kdyz asi chapu, proc se to nekomu zdalo zbytecny :)

Reply:
ok, tak ty testy předělám v dalším PR ;) s touhle změnou to stejně moc nesouvisí

Reply:
jj, akorát netušíš jestli ta změna funguje nebo ne IMHO ;)

Reply:
https://github.com/keboola/php-datatypes/pull/30#pullrequestreview-384542391